cmd/agent/common/import.go

```diff
@@ -100,7 +100,7 @@ func ImportConfig(oldConfigDir string, newConfigDir string, force bool) error {
 
 	// move existing config files to the new configuration directory
 	files, err := ioutil.ReadDir(filepath.Join(oldConfigDir, "conf.d"))
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
what's this change for?

Should we log something (or return an error) when the directory doesn't exist, and why/why not?

Reply:
We might not want to return an err in this case because there may be somewhere else (next loop around line 155) where conf files are present. Therefore, even if loop 155 has nothing to process there is still the main configuration which is imported at the start of this method.
Let me know if you think that not any conf.d directories (no auto conf files and no files in conf.d) is a case where we should errorred, I'll change it accordingly.

Reply:
ok, I think I understand, so this is meant to handle the following scenario correct?
1. the user installs Agent 5 and doesn't create/modify integration configs (just `datadog.conf` and the default integration configs)
2. the user upgrades to Agent 6
3. the user uses the `import` command to import the Agent 5's config: because of `1.` the Agent 5's `conf.d` directory doesn't exist, but we still want the command to import `datadog.conf` without errors.

Is my understanding correct?

In this case I completely agree that the import command should not error out, good that this fixes it 👍 Still, it'd be nice to print a small informational message about this, e.g.:

```
No `conf.d` directory found, skipping import of integration configs.
```

(let me know what you think, and in any case feel free to address this in a separate PR)

Reply:
one more thing: this specific change justifies a small release note, so please add one :)

Reply:
Yep that's indeed for this special case. I'll make the change directly in this PR + the reno.